### DISCUSSION TITLE
Decision to use camelCase vs kebab-case for tags?

### DISCUSSION THREAD

**Discussion by h34lthbybr33** [avitorio/outstatic#252] [General]
May I ask the decision to use [camel case](https://github.com/avitorio/outstatic/blob/canary/packages/outstatic/src/components/TagInput/index.tsx#L56) vs [kebab case](https://github.com/avitorio/outstatic/blob/canary/packages/outstatic/src/components/DocumentTitleInput/index.tsx#L49) when it comes to tags? Seems odd to have a blog-post-slug not match a blogPostTag?

Any thoughts from others on this?

**Comment by avitorio** [comment]
Hey,

It's been a while but I believe this was done because tags are stored in the schema.json, together with the custom fields. And camel case is the convention for keys in json. So all fields are camel case, and that was extended to tags.

**Comment by h34lthbybr33** [comment]
I get the json notation, but they're stored as `{"label":"Two
Words","value":"twoWords"}` so I get it but I don't. You could change to
kebab and wouldn't functionally change things, that I can tell?

**Reply by avitorio** [reply]
You're correct. We could use kebab case there, but making the switch now would probably break sites that use the current convention.

I am not against changing it, if there's community support behind the idea, but it would have to be done on a major version release with additional docs added for the breaking changes.
